### PR TITLE
[#6815] restyle sent-transaction screen

### LINKED ISSUE #6815: Restyle wallet transaction sent screen
Restyle the final wallet screen so resembles the final transaction sent screen in the new wallet flow.

Figma Wallet Setup
https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction?node-id=0%3A1

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### PR DESCRIPTION
**Pull request by bhauman**
fixes #6815 

### Summary:

This restyles the Wallet's final transaction sent screen. This was done according the `Transaction Sent` screen in the following figma.

https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction?node-id=0%3A1

### Review notes:

I have taken much care in this commit to make the code clear and meaningful.  The reviewer will see that I have inlined styles in quite a few instances. It is my opinion that abstractions like `component/flex` which means `{:flex 1}` are detrimental to the understanding of the code. When abstracting and naming things they need to merit abstraction and provide value  (i.e `map` is a good abstraction and `components/flex` isn't). If an abstraction is just a name that doesn't mean anything, all that name does is remove meaning from the immediate and it obfuscates it.

Inlining simple styles so that it helps the reader understand why this `react/view` component is being used is very very helpful.

I have also inlined style that creates a graphical figure because that is the full meaning of the code to abstract that removes meaning from the location where you are actually looking for it.

If a style removes meaning from the view or if it is complex and reused I say yes abstract it, and if you do abstract it give it a meaningful name.

### Testing notes:

Just ensure that the Transaction sent screen is displayed as it is in the figma file.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted (optional)
**Functional**
- wallet / transactions

### Steps to test:
- Open Status
- Send a transaction and view the final fabulous Transaction Sent screen
- Hit the done button and see that it behaves correctly

status: ready

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### CODE REVIEW

**Comment by bhauman** [comment]
@antdanchenko This PR has UI content changes. 

The final button now says `Done` instead of `Got it`. I will attach some screen shots.

**Comment by bhauman** [comment]
IOS screen shot
simulator screen shot - iphone 7 - 2018-11-21 at 12 38 11

**Comment by bhauman** [comment]
Android Nexus 5:
android-screen

File: src/status_im/ui/screens/wallet/transaction_sent/views.cljs [hunk 1]
```diff
@@ -3,38 +3,71 @@
...
+               :align-items      :center
+               :justify-content  :center}
+   [react/view {:background-color colors/white
+                :width            80
+                :height           80
+                :border-radius    41
+                :shadowRadius     4
+                :shadow-offset    {:width 0 :height 2}
+                :shadowOpacity    0.8
+                :shadowColor      "rgba(43, 59, 71, 0.12)"
```
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/transaction_sent/views.cljs]: ocd nitpick, we should only use kebab-case, e.g `shadow-color`

File: src/status_im/ui/screens/wallet/transaction_sent/views.cljs [hunk 2]
```diff
@@ -3,38 +3,71 @@
...
+                :font                (if platform/android? :medium :default)
+                :accessibility-label :transaction-sent-text}
+    (i18n/label :t/transaction-sent)]
+   [react/i18n-text {:style styles/transaction-sent-description
+                     :key   :transaction-description}]])
+
+(defn- bottom-action-button [on-next]
+  [react/touchable-highlight {:on-press on-next
+                              :style {:border-top-width 1
+                                      :border-color     "rgba(255, 255, 255, 0.1)"}
```
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/transaction_sent/views.cljs]: we have a color in the pallete for this - https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/colors.cljs#L24

same thing for other colors, if they're in the palette

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/wallet/transaction_sent/views.cljs (hunk 2 above)
Comment (bhauman) [reply src/status_im/ui/screens/wallet/transaction_sent/views.cljs]: fixed!

**Review by yenda — APPROVED** [review]

**Comment by bhauman** [comment]
@asemiankevich and this one? :)

**Comment by asemiankevich** [comment]
@bhauman on it

**Comment by asemiankevich** [comment]
looks good! @yevh-berdnyk @hetvart `Got it` should be replaced with `Done` in tests 🍪 

image

image

upd: seems locator is not changed btw :D

**Pull request merged**